server/locale/DE/GPL/mutations/gpl_gtz.js

```diff
@@ -6,9 +6,61 @@ import {
     logoInlineAddRatenzahlungAfterPayPal,
     middleAndSmallestFallback
 } from './mediaQueries';
-import { textLogoMutations } from '../../../../message/logoMutations';
+import { flexLogoMutations, textLogoMutations } from '../../../../message/logoMutations';
+
+const headlineBreaks = [
+    {
+        sizes: ['xsmall'],
+        breaks: ['mit']
+    },
+    {
+        sizes: ['medium'],
+        breaks: ['monatlichen']
+    }
+].reduce((acc, item) => {
+    const { sizes, breaks } = item;
+    sizes.forEach(size => {
+        acc.push({
+            tag: size,
+            br: breaks
+        });
+    });
+    return acc;
+}, []);
+
+const flex = [
+    [
+        'default',
+        {
+            logo: Logo.PP_PAYPAL.WHITE,
+            headline: [
+                {
+                    tag: 'xsmall'
+                },
+                {
+                    tag: 'medium'
+                }
+            ],
+            disclaimer: ['default']
+        }
+    ],
+    [
+        'ratio:20x1',
+        {
+            headline: [...headlineBreaks]
+        }
+    ],
+    [
+        'ratio:8x1',
+        {
+            headline: [...headlineBreaks]
+        }
```

Comment:
See above explanation. 
```suggestion
        'ratio:8x1',
        {
            styles: [addPeriod()],
            headline: [...headlineBreaks]
        }
```